### PR TITLE
Fix discrepancies between pontoon and rvierbridge

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
Bugfixes "Fix discrepancies between pontoon and rvierbridge"

#### Purpose of change

- give them slightly different descriptions
- give them the same recipe
- fix wrong smash result in riverbridge

#### Testing
Load game without error
Build a pontoon over a pond
Build a riverbridge over a river
No problem

### CODE REVIEW

File: data/json/furniture_and_terrain/terrain-bridges-docks.json [hunk 1]
```diff
@@ -118,7 +118,7 @@
     "type": "terrain",
     "id": "t_pontoon_dp",
     "name": "pontoon bridge",
-    "description": "A floating temporary bridge, like the ones army used to make to cross rivers.",
+    "description": "A floating temporary bridge over still water, like the ones army used to make to cross lakes.",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-bridges-docks.json:121]:
```suggestion
    "description": "A floating temporary bridge over still water, like the ones armies used to make to cross lakes.",
```
Or "the army".

File: data/json/furniture_and_terrain/terrain-bridges-docks.json [hunk 2]
```diff
@@ -141,7 +141,7 @@
     "type": "terrain",
     "id": "t_riverbridge_dp",
     "name": "river bridge",
-    "description": "A floating temporary bridge, like the ones army used to make to cross rivers.",
+    "description": "A floating temporary bridge over flowing water, like the ones army used to make to cross rivers.",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-bridges-docks.json:144]:
```suggestion
    "description": "A floating temporary bridge over flowing water, like the ones armies used to make to cross rivers.",
```
Or "the army".

**Comment by Fris0uman** [comment]
sorry accidental close ^^"

**Comment by actual-nh** [comment]
Oh - BTW - summary typo fix for the changelog:
`Bugfixes "Fix discrepancies between pontoon and riverbridge"`

**Pull request merged**